test/components/selectionBoxLayerTests.ts

```diff
@@ -118,18 +118,7 @@ describe("Interactive Components", () => {
         assert.strictEqual(sbl.xScale(), xScale, "The getter returns the correct scale");
         sbl.renderTo(svg);
 
-        let topLeft: Plottable.Point = {
-          x: 0,
-          y: 0
-        };
-        let bottomRight: Plottable.Point = {
-          x: 250,
-          y: 300
-        };
-        sbl.bounds({
-          topLeft: topLeft,
-          bottomRight: bottomRight
-        });
+        sbl.xExtent([0, 1000]);
```

Comment:
This looks like a change in behavior. Previously we were setting the bounds, right?